eng/Packages.Data.props

```diff
@@ -101,8 +101,15 @@
     <PackageReference Update="System.Text.Json" Version="6.0.10" />
     <PackageReference Update="System.Text.Encodings.Web" Version="6.0.0" />
     <PackageReference Update="System.ValueTuple" Version="4.5.0" />
-    <PackageReference Update="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" />
+    <PackageReference Update="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" />
```

Comment:
This one needs to be bumped because of Microsoft.Extensions.Logging

Reply:
Why?  Microsoft.Extensions.Logging says it is compatible with net461: https://apisof.net/catalog/1587d1bdfa66eaa6a3d734408681d41b?fx=net461